README.md

```diff
@@ -58,14 +58,20 @@ performance, small size and all the features listed below.
 ## Example
 
 ```javascript
-var snabbdom = require('snabbdom');
-var patch = snabbdom.init([ // Init patch function with chosen modules
-  require('snabbdom/modules/class').default, // makes it easy to toggle classes
-  require('snabbdom/modules/props').default, // for setting properties on DOM elements
-  require('snabbdom/modules/style').default, // handles styling on elements with support for animations
-  require('snabbdom/modules/eventlisteners').default, // attaches event listeners
+import { init } from 'snabbdom';
+import clazz from 'snabbdom/class';
```

Comment:
Why still split? With tree shaking we could have everything just as snabbdom package

Reply:
pure es modules is potentially enabling a bundless workflow; tree shaking is an artifact of bundling.

Reply:
If we place everything in one module, we gain a more concise import (1 statement with multiple names). But we lose "tree shaking" in the case of not using a bundler. I would pay the price of the multiple import statements for split modules without a bundler.

Reply:
And shorter files...